src/QuantType.js

```diff
@@ -22,6 +22,10 @@ const defaults = {
     assay: { type: 'Quant Type', version: 'v1.0' },
     outlier: { type: 'standard', threshold: '1' },
     fields: ['a', 'b', 'c', 'd', 'e', 'f'],
+    warningThreshold: {
+      value: 5,
+      message: 'Warning: This is a test warning.',
```

Comment:
Are we going to wait for UAT to decide what the warning message should be?

Reply:
So this is just in the 'defaults' section. I'm using this text in the tests.

The real message is in the quantTypes.json.

Let me know if that seems wrong though, as I wasn't quite sure if these defaults were used for anything other than test data.